standard/standard/annex-glossary.adoc

```diff
@@ -1,97 +1,132 @@
-[appendix]
-:appendix-caption: Annex 
+[[annex-glossary]]
+[appendix,obligation=informative]
 == Glossary
 
-The following terms and concepts have appeared in the discussion of the use cases and design of the GeoPose standard but not part of the terms defined in Terms and Definitions. These
+The following terms and concepts have appeared in the discussion of the use cases and
+design of the GeoPose standard but not part of the terms defined in Terms and
+Definitions. These
 
-*** *Acceleration:* 
-The time rate of change of velocity.
+* *Acceleration*: The time rate of change of velocity.
```

Comment:
All terms in the OGC Annex Glossary must be formatted as Terms and definitions just like Clause 4 here.

Reply:
@ronaldtse Terms are currently not formatted as usually in Clause 4. Should I apply standard markup for terms and definitions in both Clause 4 and Glossary?